hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/replication/VerifyReplication.java

```diff
@@ -575,11 +632,21 @@ public boolean doCommandLine(final String[] args) {
           continue;
         }
 
-        final String sleepToReCompareKey = "--recomparesleep=";
+        final String deprecatedSleepToReCompareKey = "--recomparesleep=";
+        final String sleepToReCompareKey = "--recompareSleep=";
+        if (cmd.startsWith(deprecatedSleepToReCompareKey)) {
+          LOG.warn("--recomparesleep is deprecated and will be removed in 4.0.0."
+            + " Use --recompareSleep instead.");
+          sleepMsBeforeReCompare =
+            Integer.parseInt(cmd.substring(deprecatedSleepToReCompareKey.length()));
+          continue;
+        }
         if (cmd.startsWith(sleepToReCompareKey)) {
-          sleepMsBeforeReCompare = Integer.parseInt(cmd.substring(sleepToReCompareKey.length()));
+          sleepMsBeforeReCompare =
+            Integer.parseInt(cmd.substring(deprecatedSleepToReCompareKey.length()));
```

Comment:
i think you meant to substring sleepToReCompareKey here. I realize it's the same length, but just to avoid errors/confusion in the future.

Reply:
ah you're totally right, this was my bad